.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -109,23 +131,18 @@ jobs:
                 title,
               });
 
-              await github.rest.issues.createComment({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                issue_number: +PR,
-                body: `Hey @${REQUESTING_USER}, I've updated #${existing.number} for you.`,
-              });
+              commentBody = `I've updated #${existing.number} for you.`;
             }
 
-      - run: |
-          MESSAGE="Hey @$REQUESTING_USER, I was unable to cherry-pick this PR."
-          MESSAGE+=$'\n\n'
-          MESSAGE+="Check the logs at: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+            core.setOutput("comment_body", commentBody);
 
-          gh pr comment "$PR" --repo ${{ github.repository }} --body "$MESSAGE"
-        if: ${{ failure() }}
-        env:
-          PR: ${{ inputs.pr || github.event.client_payload.pr }}
-          TARGET_BRANCH: ${{ inputs.target_branch || github.event.client_payload.target_branch }}
-          REQUESTING_USER: ${{ inputs.requesting_user || github.event.client_payload.requesting_user }}
-          GH_TOKEN: ${{ secrets.TS_BOT_GITHUB_TOKEN }}
+      - uses: microsoft/TypeScript/.github/actions/post-workflow-result@main
```

Comment:
I feel sorta gross writing this; the reason this isn't just `./.github/actions/post-workflow-result` alone is because most of these workflows don't actually clone TS or clone a random version (e.g. to make a release branch).

Maybe this is a reason why I should just make a new repo for this bot stuff instead.